src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -109,78 +125,525 @@ describe('Alerts (Unit)', () => {
-          'transaction',
-          alertTransactionBuilder()
-            .with('to', delayModifier)
-            .with('logs', [
-              alertLogBuilder()
-                .with('address', delayModifier)
-                .with('data', transactionAddedEvent.data)
-                .with('topics', transactionAddedEvent.topics)
-                .build(),
-            ])
-            .build(),
-        )
-        .with('event_type', EventType.ALERT)
-        .build();
-      const timestamp = Date.now().toString();
-      const signature = fakeTenderlySignature({
-        signingKey,
-        alert,
-        timestamp,
+    describe('it notifies about a valid transaction attempt', () => {
+      it('notifies about addOwnerWithThreshold attempts', async () => {
+        const chain = chainBuilder().build();
+        const delayModifier = faker.finance.ethereumAddress();
+        const safe = safeBuilder().with('modules', [delayModifier]).build();
+
+        const addOwnerWithThreshold = addOwnerWithThresholdEncoder();
+        const { threshold, owner } = addOwnerWithThreshold.build();
+        const transactionAddedEvent = transactionAddedEventBuilder()
+          .with('data', addOwnerWithThreshold.encode())
+          .with('to', safe.address)
+          .build();
+
+        const alert = alertBuilder()
+          .with(
+            'transaction',
+            alertTransactionBuilder()
+              .with('to', delayModifier)
+              .with('logs', [
+                alertLogBuilder()
+                  .with('address', delayModifier)
+                  .with('data', transactionAddedEvent.data)
+                  .with('topics', transactionAddedEvent.topics)
+                  .build(),
+              ])
+              .with('network', chain.chainId)
+              .build(),
+          )
+          .with('event_type', EventType.ALERT)
+          .build();
+        const timestamp = Date.now().toString();
+        const signature = fakeTenderlySignature({
+          signingKey,
+          alert,
+          timestamp,
+        });
+        const verifiedSignerEmails = [{ email: faker.internet.email() }];
+        emailDataSource.getVerifiedSignerEmailsBySafeAddress.mockResolvedValue(
+          verifiedSignerEmails,
+        );
+
+        networkService.get.mockImplementation((url) => {
+          switch (url) {
+            case `${safeConfigUrl}/api/v1/chains/${chain.chainId}`:
+              return Promise.resolve({ data: chain });
+            case `${chain.transactionService}/api/v1/safes/${getAddress(
+              safe.address,
+            )}`:
+              return Promise.resolve({ data: safe });
+            default:
+              return Promise.reject(`No matching rule for url: ${url}`);
+          }
+        });
+
+        await request(app.getHttpServer())
+          .post('/v1/alerts')
+          .set('x-tenderly-signature', signature)
+          .set('date', timestamp)
+          .send(alert)
+          .expect(200)
```

Comment:
Out of scope for this PR but [HTTP 202](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202) better represents the outcome of the alerts route.

Reply:
I'll address this in a follow up PR.

Reply:
I've created [a separate PR](https://github.com/safe-global/safe-client-gateway/pull/948) that tackles this. I will update the tests accordingly when merging.